static/bidder-info/rxnetwork.yaml

```diff
@@ -0,0 +1,2 @@
+aliasOf: adkernel
+gvlVendorID: 14
```

Comment:
Checking to ensure the use of adkernel's TCF vendor code is appropriate within the context of the RxNetwork alias.

Reply:
I've intentionally specified Adkernel's gvlid to claim GDPR support. I saw a similar approach in other aliased adaptors.

Reply:
There's no need to specify it here. It will be automatically picked up with the alias.

Reply:
@SyntaxNode ok, updated